ports/atmel-samd/boards/grandcentral_m4_express/pins.c

```diff
@@ -137,6 +137,7 @@ STATIC const mp_rom_map_elem_t board_global_dict_table[] = {
 
     { MP_OBJ_NEW_QSTR(MP_QSTR_NEOPIXEL), MP_ROM_PTR(&pin_PC24) },
 
+    { MP_OBJ_NEW_QSTR(MP_QSTR_LED), MP_ROM_PTR(&pin_PB01) },
```

Comment:
Please move this up to where `D13` is defined, and put blank lines around D13 and LED, to indicate that it's a pair of aliases. Thanks.

Reply:
Done